### PR TITLE
added routes for job list & job details

### PR DESCRIPTION
**Pull request by nspc97**
Unfortunately, we don't know why but the changes from the last pull request (added fixtures) are also displayed in the new pull request, but we have also added the new changes with routes for jobs_list (/api/activities) and for job details (/api/activities/{id})

### CODE REVIEW

File: src/Entity/Technology.php [hunk 1]
```diff
@@ -5,16 +5,21 @@
...
+ * @Serializer\ExclusionPolicy("all")
  */
 class Technology
 {
     /**
      * @ORM\Id()
      * @ORM\GeneratedValue()
      * @ORM\Column(type="integer")
+     * @Serializer\Expose()
+     * @Groups({"ActivityDetails"})
```
Comment (SpartakusMd) [inline src/Entity/Technology.php]: Why have you made only this field available in the ActivityDetails? What about the others?

File: src/Entity/Type.php [hunk 2]
```diff
@@ -5,16 +5,21 @@
...
+ * @Serializer\ExclusionPolicy("all")
  */
 class Type
 {
     /**
      * @ORM\Id()
      * @ORM\GeneratedValue()
      * @ORM\Column(type="integer")
+     * @Serializer\Expose()
+     * @Groups({"ActivityDetails"})
```
Comment (SpartakusMd) [inline src/Entity/Type.php]: Why have you made only this field available in the ActivityDetails? What about the others?

File: src/Entity/User.php [hunk 3]
```diff
@@ -21,11 +24,14 @@ class User
      * @ORM\Id()
      * @ORM\GeneratedValue()
      * @ORM\Column(type="integer")
+     * @Serializer\Expose()
+     * @Groups({"ActivityDetails"})
```
Comment (SpartakusMd) [inline src/Entity/User.php]: Why have you made only this field available in the ActivityDetails? What about the others?

File: templates/api/activity.html.twig [hunk 4]
```diff
@@ -0,0 +1,16 @@
+{# templates/api/activities.html.twig #}
```
Comment (SpartakusMd) [inline templates/api/activity.html.twig]: Please remove this file or ignore it.

**Review by SpartakusMd — CHANGES_REQUESTED** [review]

File: composer.json [hunk 5]
```diff
@@ -7,6 +7,9 @@
     "ext-iconv": "*",
     "friendsofsymfony/rest-bundle": "^2.5",
     "fzaninotto/faker": "^1.8",
+    "jms/serializer-bundle": "^3.3",
+    "knpuniversity/oauth2-client-bundle": "^1.28",
```
Comment (SpartakusMd) [inline composer.json]: You should not have changes from two tasks on the same brach.

**Review by SpartakusMd — CHANGES_REQUESTED** [review]

**Comment by SpartakusMd** [comment]
Please be specific about the comments. Don't use the same comment twice. Describe the changes.

File: composer.json (hunk 5 above)
Comment (nspc97) [reply composer.json]: @SpartakusMd 
 Jms/serializer is used for the task with job details/list, so it's related to this task.
But in the case of oauth2 bundle...we need to simply remove this row, or we can just ignore it for this pull request?

Comment (SpartakusMd) [reply composer.json]: By the way, nelmio/api-doc-bundle is also from another task :)

**Pull request merged**